extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -324,7 +324,9 @@ export class AmpAutocomplete extends AMP.BaseElement {
     this.minChars_ = this.element.hasAttribute('min-characters')
       ? parseInt(this.element.getAttribute('min-characters'), 10)
       : 1;
-    this.maxEntries_ = this.element.hasAttribute('max-entries')
+    this.maxItems_ = this.element.hasAttribute('max-items')
+      ? parseInt(this.element.getAttribute('max-items'), 10)
+      : this.element.hasAttribute('max-entries')
```

Comment:
Wrap the entire ternary in a `parseInt`? `max-entries` needs to be parsed as well.

Reply:
Actually I can't read, didn't notice the sequence. :) 

Maybe simplify?

```js
const maxItems = this.element.getAttribute('max-items') ||
  this.element.getAttribute('max-entries');
this.maxItems_  = maxItems ? parseInt(maxItems, 10) : null;
```